packages/manager/apps/web-ongoing-operations/src/pages/dashboard/dns/Dns.tsx

```diff
@@ -10,14 +10,21 @@ import {
 import { getmeTaskDnsList } from '@/data/api/web-ongoing-operations';
 
 export default function Domain() {
-  const { data: dnsList, isLoading } = useQuery<TOngoingOperationsData>({
+  const fetchDnsList = async () => {
+    const response = await getmeTaskDnsList();
+    return response.data as TOngoingOperations[];
+  };
```

Comment:
I don't understand why you need to create a function to only get the data property, previous implementation was better

Or you should use select from tanstack query or useMemo from react after the call